modules/claims_api/app/swagger/claims_api/v2/swagger_root.rb

```diff
@@ -30,6 +30,14 @@ class SwaggerRoot
           key :description, 'Allows authenticated and authorized users to access claims data for a given Veteran. No data is returned if the user is not authenticated and authorized.'
         end
 
+        server do
```

Comment:
i would almost just remove sandbox and prod entirely from this, so the user cant accidentally pick a different environment.
we're going to have to edit this list before it goes all the way out to production anyways, to remove dev from this list.

Reply:
Oh, good call!
[Removed here](https://github.com/department-of-veterans-affairs/vets-api/pull/7045/commits/0fdb563f08aa3af8cce7c5c399a3228967cd92d0).